### ISSUE TITLE
Import statements are broken in documentation and help commands in the Terraform provider are outdated.

### ISSUE THREAD

**Issue by jonathanmadeley** [mongodb/terraform-provider-mongodbatlas#956]
### Version
```hcl
Terraform v1.1.6
+ provider registry.terraform.io/mongodb/mongodbatlas v1.6.0
```

### Steps to Reproduce
Please list the full steps required to reproduce the issue, for example:
1. `terraform init`
2. Create a `mongodbatlas_event_trigger`.
3. Try to import an existing trigger using the documentation on Terraform. (https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/event_trigger#import)
4. The documentation is incorrect, only the example is correct. Running "project_id--app_id-trigger_id" produces the error: `import format error: to import a MongoDB Event Trigger, use the format {project_id}-{app_id}-{trigger_id} ` which again is the incorrect format.
5. The example on the documentation (https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/event_trigger#import) also states one should use the text `app_id`. This produces a 404 as this is not what the Mongo API expects. Using the numerical ID works but this is not mentioned in the documentation.

**Comment by martinstibbe** [comment]
@jonathanmadeley verified what you are seeing resourceMongoDBAtlasEventTriggerImportState is using this -- as a delimiter strings.Split(d.Id(), "--") and docs and error messages are not lined up as well will correct this under engineering ticket [INTMDB-513](https://jira.mongodb.org/browse/INTMDB-513)

**Comment by Charoenchai2518** [comment]
111

**Comment by Zuhairahmed** [comment]
@jonathanmadeley @Charoenchai2518 as an update, this issue has been picked up in the next sprint cycle and you can expect fix to be included as part of v1.8.1 release next month. will let you know when latest release has been published to the Terraform Registry.

**Comment by Zuhairahmed** [comment]
@jonathanmadeley @Charoenchai2518  v1.8.1 has been published to the [Terraform Public Registry](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest)! take a spin and let us know if you need anything else.
